portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,35 +52,56 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
     response.getWriter().println("Hello Chiamaka");
 
+    **/
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    PreparedQuery results = datastore.prepare(query);
+
+    int maxComments = 10;
+
+    try {
+    String getMax = request.getParameter("max");
+    maxComments = Integer.parseInt(getMax);
+    } catch (NumberFormatException e){
+    maxComments = 10;
```

Comment:
This is redundant, you already initialized it to 10

Reply:
Resolving because removed redundancy